builtin/mainmenu/init.lua

```diff
@@ -24,6 +24,15 @@ mt_color_dark_green = "#25C191"
 
 local menupath = core.get_mainmenu_path()
 local basepath = core.get_builtin_path()
+local menustylesetting = core.settings:get("main_menu_style")
+local menustyle = menustylesetting
+if menustyle == "default" then
+	if PLATFORM == "Android" then
+		menustyle = "simple"
+	else
+		menustyle = "full"
+	end
```

Comment:
`menustyle = PLATFORM == "Android" and "simple" or "full"`

Reply:
Is that similar to a ternary operator?

Reply:
Didn't know about that, thanks.